brd-ios/breadwallet/src/Redesign/Scenes/Buy/CardSelection/CardSelectionViewController.swift

```diff
@@ -52,7 +52,8 @@ class CardSelectionViewController: ItemSelectionViewController {
         }
         
         cell.setup { view in
-            view.configure(with: .init())
+            let config = CardSelectionConfiguration(title: .init(font: Fonts.Title.six, textColor: LightColors.Icons.one))
```

Comment:
do we have a Presenter for this VC?

Reply:
here we are reusing the ItemSelectionViewController, that's why the config is setup here here